packages/cli/src/utils/deploy-v2.ts

```diff
@@ -365,3 +419,12 @@ function toBytes16(input: string) {
   }
   return result;
 }
+
+function toResourceSelector(namespace: string, file: string): Uint8Array {
+  const namespaceBytes = toBytes16(namespace);
+  const fileBytes = toBytes16(file);
+  const result = new Uint8Array(32);
+  result.set(namespaceBytes);
+  result.set(fileBytes, 16);
+  return result;
+}
```

Comment:
We should use TableId from `@latticexyz/utils` here, but I wasn't able to import it, my guess is because utils doesn't use `.js` imports, so I'll leave that refactor for later

Reply:
yeah before cli ESM migration utils just couldn't be used by cli. Now with some refactoring they probably can

Reply:
can we add a TODO + issue?

Reply:
#499